content/en/guides/employee-engagement.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Employee Open Source Engagement
+---
+
+This guide is meant for people who make open source contributions as part of their work for an employer. Companies and other organizations engaging with open source communities and enabling their employees to work with open source projects is a very direct form of collaboration. This can create a lot of value for all participants, the organization contributing, the open source project receiving the contributions, the people contributing, and the users of the project. But there also are pitfalls which can make contributions difficult and ineffective, cause conflict or let them fail to achieve their desired result.
+
+So how do you interact with a community? How do you make sure you are respecting the needs of all participants, including the needs of your organization? How do you make sure that your participation is valuable?
+
+There are expectations and rules, there is culture and conventions. Not all of this is obvious. You will need to follow organization-internal rules and procedures. That is covered in another guide, the [Guide to Outbound Open Source Software](https://todogroup.org/guides/outbound-oss/). This guide here will focus on the organization-external perspective. To engage with the open source community in a respectful, transparent, and productive way, we call "authentic participation".
+
+Here we lay out the six principles of authentic participation:
+
+**Authentic participation...**
+
+* [Starts Early.](#principle-1-starts-early)
+* [Puts the Community First.](#principle-2-puts-the-community-first)
+* [Starts With Listening.](#principle-3-starts-with-listening)
```

Comment:
From today's calls discussion:

Additional principle: "Continues by Participating."

Asking question, being a voice in the community, answering to issues, see where you can help (note that helping can be a big step for a new contributor) ...

Reply:
Addressed by section with new principle.